src/core/dev/hw_queue_tx.cpp

```diff
@@ -996,6 +991,7 @@ void hw_queue_tx::nvme_set_static_context(xlio_tis *tis, uint32_t config)
 void hw_queue_tx::nvme_set_progress_context(xlio_tis *tis, uint32_t tcp_seqno)
 {
     auto *wqe = reinterpret_cast<mlx5e_set_nvmeotcp_progress_params_wqe *>(m_sq_wqe_hot);
+    memset(wqe, 0, sizeof(mlx5_eth_wqe));
```

Comment:
similarly to the `nvme_set_static_context()` comment, `nvme_fill_progress_wqe()` clears the wqe inside. However, in this case, the wqe is 32 bytes. I'd suggest to:
* remove the memset inside the nested function
* replace sizeof(mlx5_eth_wqe) with WQEBB constant. Ethernet wqe can be confusing in case of non-send operations. For more readability, this can even be `constexpr` `max<size_t>(WQEBB, sizeof(struct_name))` - it is expected to be optimized in compile time.

Reply:
Done